src/widget/wsearchlineedit.h

```diff
@@ -22,9 +24,13 @@ class WSearchLineEdit : public QComboBox, public WBaseWidget {
     static constexpr int kMaxDebouncingTimeoutMillis = 9999;
     static constexpr int kSaveTimeoutMillis = 5000;
     static constexpr int kMaxSearchEntries = 50;
+    static constexpr bool kEnableSearchCompletionsByDefault = true;
+    static constexpr bool kEnableSearchHistoryShortcutsByDefault = true;
```

Comment:
```suggestion
    static constexpr bool kEnableSearchCompletionsByDefault = true;
    static constexpr bool kEnableSearchHistoryShortcutsByDefault = true;
```
because 'search' is redundant e.g. when getting it from within the preferences `WSearchLineEdit::kEnableCompletionsByDefault`